examples/simple_dashboard/Pipfile

```diff
@@ -0,0 +1,16 @@
+[[source]]
+url = "https://pypi.org/simple"
+verify_ssl = true
+name = "pypi"
+
+[packages]
+hikari = "*"
+rillrate = "*"
+
+[dev-packages]
+
+[requires]
+python_version = "3.9"
```

Comment:
We support python3.8 (which rillrate seems to support too). Might be worth removing the Pipfiles and to keep it as simple as possible, the requirements file should cover this.

*Sorry forgot to comment on this one, I promise its the last :sweat_smile:*

Reply:
decided to keep it but lower the version to 3.8 on #0fe94aa0

Reply:
Requirements.txt already covers this case tho. Mostly just to keep consistent as well as we dont make use of Pipfiles throught he project

Reply:
fixed on #dd6aab3f